### PR TITLE
Add MUI as a peer dependency

### PR DESCRIPTION
**Pull request by Honglin-Lu**

### CODE REVIEW

**Review by wenfeng-xing — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -14,11 +14,21 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "@mui/icons-material": "^5.10.6"
```
Comment (chuck-adaptive) [inline package.json:17]: I am not sure we'll use these icons. I'm also thinking they should be a peerDependency if/when we do

**Review by chuck-adaptive — APPROVED** [review]
Minor note on the icons - I may be missing something

If it can be moved let's do it and ✅

File: package.json (hunk 1 above)
Comment (Honglin-Lu) [reply package.json:17]: Ok, I'll remove it now and add it later if we need .

Comment (Honglin-Lu) [reply package.json:17]: I'll move it !

**Pull request merged**